disnake/interactions/base.py

```diff
@@ -397,12 +397,15 @@ async def edit_original_message(
             Files will be appended to the message, see the ``attachments`` parameter
             to remove/replace existing files.
         attachments: List[:class:`Attachment`]
```

Comment:
```suggestion
        attachments: Optional[List[:class:`Attachment`]]
```
same thing for other `attachments` parameters

Reply:
https://github.com/DisnakeDev/disnake/pull/457/commits/3dc30b1af1fdbfd9fddbd3ab07370b36cae42029